AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -1414,3 +1575,11 @@ msgstr "Your username, if you have forgotten it, is:"
 #: templates/registration/password_reset_email.html:10
 msgid "Obrigado por usar o InfoDengue"
 msgstr "Thank you for use InfoDengue"
+
+#~ msgid ""
```

Comment:
remove this kind of comments `#~ ......` this is related to old translations that is not used anymore

Reply:
good

Reply:
ok